### PR TITLE
mendel-manifest-uglify to solve some compression inconsistencies

### PR DESCRIPTION
**Pull request by irae**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: packages/mendel-manifest-uglify/manifest-uglify.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+function manifestUglify(manifests, options, next) {
+    var optBundles = [].concat(options.bundles).filter(Boolean);
+    var uglifyOptions = Object.assign({
+        compress: true,
+        mangle: true,
+    }, options.uglifyOptions, {
+        fromString: true
+    });
+
+    function wichManifests(manifest) {
```
Comment (stephanwlee) [inline packages/mendel-manifest-uglify/manifest-uglify.js]: seems like a small typo.

File: test/manifest-uglify.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            // just re-using, important part is that are some files there
+            bundleName: 'uncompressed',
+            manifest: 'one-file.manifest.json'
+        }],
+    }, function(error) {
+        t.error(error);
+        var mangled = require(
+            path.join(copySamples, 'one-file.manifest.json'));
+        var extraMangleSize = mangled.bundles[0].data[0].source.length;
+        t.assert(extraMangleSize/simpleCompressionSize < 0.8, 'compressed with options');
```
Comment (stephanwlee) [inline test/manifest-uglify.js]: nice!

File: packages/mendel-manifest-uglify/manifest-uglify.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   See the accompanying LICENSE file for terms. */
+
+var UglifyJS = require("uglify-js");
+var debug = require("debug")('mendel-manifest-uglify');
+
+module.exports = manifestUglify;
+
+function manifestUglify(manifests, options, next) {
+    var optBundles = [].concat(options.bundles).filter(Boolean);
+    var uglifyOptions = Object.assign({
```
Comment (stephanwlee) [inline packages/mendel-manifest-uglify/manifest-uglify.js]: this is a different default option than the uglify itself. Want to document this a little bit

File: test/manifest-uglify.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    });
+});
+
+
+
+function copyRecursiveSync(src, dest) {
+  var exists = fs.existsSync(src);
```
Comment (stephanwlee) [inline test/manifest-uglify.js]: `fs.exists` has a deprecation warn fyi.

File: packages/mendel-manifest-uglify/manifest-uglify.js (hunk 3 above)
Comment (irae) [inline packages/mendel-manifest-uglify/manifest-uglify.js]: I think it is the same, actually. I just put it there to be more explicit about it. It is at least the options for uglifyify. I'll add some comment about it anyway, it doesn't hurt.

File: test/manifest-uglify.js (hunk 4 above)
Comment (irae) [inline test/manifest-uglify.js]:
```
¯\_(ツ)_/¯ 
```

**Pull request merged**